.isort.cfg

```diff
@@ -0,0 +1,5 @@
+[settings]
+line_length = 120
+src_paths = doctr,tests
+skip_glob=**/__init__.py
+known_third_party=tensorflow,torch,torchvision,wandb,fastprogress
```

Comment:
Hhat is this line for? 
I am not familiar with .isort.cfg

Reply:
It's similar to flake8 but specify to import ordering